### DISCUSSION TITLE
cannot auth/login w/ ldap (FreeIPA)

### DISCUSSION THREAD

**Discussion by slekkus75** [ansibleguy76/ansibleforms#168] [Q&A] [answered]
Hi, not sure what I do wrong, but cannot get ldap authenctication working *user not found or usernameAttribute is wrong*
LDAP is FreeIPA with AD one way trust. AD as local domain users do not auth. 

These are the settings. 
![AnsibleForms](https://github.com/ansibleguy76/ansibleforms/assets/157806656/268896ff-9e4e-4b00-8c69-5eae72e42ffc)

```
- name: ldap_test
  groups:
    - ldap/adusers_posix
    - ldap/adusers_external
    - ldap/testlocalusers
 ```

**Comment by slekkus75** [comment]
Got it working for the local users by changing "user attribute" to uid
This is the schema 389 ldap / Freeipa uses.

**Reply by ansibleguy76** [reply]
so it's resolved ?

**Reply by slekkus75** [reply, marked as answer]
yes, resolved.
for anyone seeking to auth active directory users mapped trough a freeipa trust. these settings work:
![afldap](https://github.com/ansibleguy76/ansibleforms/assets/157806656/bfdfec44-d9a3-45d0-8827-a1eb82e522de)
